### DISCUSSION TITLE
Syntax highlighting on embedded gist fails

### DISCUSSION THREAD

**Discussion by ahandsel** [community/community#24337] [Apps, API and Webhooks]
<p>Thank you for your reply and for checking the website nethgato!</p>
<p>The problem we are facing is that we are unable to highlight the code when we are selecting the it.<br>
This gif shows what I mean.<br>
First, the regular text highlights as normal when selecting it to copy-&amp;-paste.<br>
However, the code inside the Gist does not highlight in the same manner.<br>
We are able to copy &amp; paste but it is not a great user experience.</p>
<p>Thank you for checking our code and glad to hear we are doing it correctly.<br>
Is this a style updating from GitHub or is this a conflict with Zendesk?</p>
<p>Thank you for your time.</p>
<p><img src="https://user-images.githubusercontent.com/30670749/181084278-87a4577c-da02-4a7a-9321-e7590b4a05b5.gif" alt="Capto_2020-12-21_15-12-40_" data-base62-sha1="mtY8aKlc5wafzCisdfTTJLPcaNA" width="640" height="378"></p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/ahandsel">@ahandsel</a> I’ve moved this post to its own thread as it no longer relates to the OP that it was spawned from.</p>
<p>Though I’m also not able to reproduce the behavior you display here and am able to highlight the code without issue:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/b/5ba740e253e69b2a88c2af40ef250c16729e1566.png" data-download-href="/uploads/short-url/d4NMTH4WsFhYz79I9AzS95ikrEG.png?dl=1" title="syntax_highlight"><img src="https://user-images.githubusercontent.com/62281706/181084280-5fbf1190-a9e2-491c-8dcd-012d902b5455.png" alt="syntax_highlight" data-base62-sha1="d4NMTH4WsFhYz79I9AzS95ikrEG" width="690" height="419" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5ba740e253e69b2a88c2af40ef250c16729e1566_2_690x419.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5ba740e253e69b2a88c2af40ef250c16729e1566_2_1035x628.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5ba740e253e69b2a88c2af40ef250c16729e1566_2_1380x838.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/b/5ba740e253e69b2a88c2af40ef250c16729e1566_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">syntax_highlight</span><span class="informations">1494×908 127 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Though I do see from your gif that you’re unable; which is really weird!</p>
<p>Can you let us know what version of Chrome you’re using? Does this happen in other browsers? On other machines? Are you logged in or out of the site?</p>

**Comment by ahandsel** [comment]
<p>Thank you so much for your support nethgato! :bowing_man:<br>
Sorry for my late response, I just got back from my holiday break.</p>
<p>The Gists on our website are indeed working!<br>
I was successful in highlighting, copying, and pasting the code hosted via GitHub Gists from our articles!</p>
<p>Looks like other people who experienced the same issue are no longer as well.</p><aside class="quote" data-post="3" data-topic="149148">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cant-select-or-copy-gist-embeds/149148/3">Can't select or copy gist embeds</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    I am no longer having this issue - it seems to have been fixed.
  </blockquote>
</aside>

<p>Sorry for troubling you during the holidays and I appreciate your quick and thorough feedback!</p>
<p>Happy holidays!</p>
